README.md

```diff
@@ -1 +1,166 @@
-# Final_capstone_Back-End
\ No newline at end of file
+# 📗 Table of Contents
```

Comment:
_note: Since this PR does not include all the content, I'll include my comments here._

## Backend project

- [ ] On your readme, the instructions about how to run your app are not correct.  A user should clone the project, navigate to the dir, run `bundle install`,  setup the database with `rails db:create, rails db:migrate, rails db:seed`, and finally run `rails server`.  Kindly add the correct instructions to run the backend API. 👍 

- [ ] On your readme, there are no instructions about how to run the tests.  Kindly add specific instructions about how to run them.

- [ ] This branch has multiple Rubocop errors.  Kindly fix them so the linter errors don't reach the repo.

That way, your project will be well documented, and comply with best practices.  💪